files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -8,6 +8,13 @@ set-kargs-hardening:
         IAEMU_NO='--append-if-missing="ia32_emulation=0"'
         echo "Disabling 32-bit support, for the next boot."
     fi
+    read -p "Do you want to disable Simultaneous Multithreading (SMT) / Hyperthreading? (This can cause a reduction in the performance of certain tasks in favor of security) [y/N]: " YES
```

Comment:
this is a bit misleading because it implies we're not already doing this on most hardware